programs/client/Client.cpp

```diff
@@ -1322,6 +1325,9 @@ void Client::readArguments(
             }
             else if (arg.starts_with("--host") || arg.starts_with("-h"))
             {
+                if (has_connection_string)
+                    throw Exception(ErrorCodes::BAD_ARGUMENTS, "Mixing connection string and --host/--port client arguments is prohibited");
```

Comment:
`Mixing a connection string and --host argument is prohibited`

Reply:
this exception is moved to ConnectionString.cpp `validateConnectionStringClientOption`